src/parser/TurtleParser.cpp

```diff
@@ -141,7 +141,7 @@ bool TurtleParser<T>::predicateSpecialA() {
   _tok.skipWhitespaceAndComments();
   if (auto [success, word] = _tok.template getNextToken<TokId::A>(); success) {
     (void)word;
-    _activePredicate = u8"<http://www.w3.org/1999/02/22-rdf-syntax-ns#type>";
+    _activePredicate = "<http://www.w3.org/1999/02/22-rdf-syntax-ns#type>"s;
```

Comment:
Why is the s needed here and not in other places?

Reply:
It is not needed. The `s` suffix makes a `std::string` constant, whereas the `""` is just a `const char*` which can also be implicitly converted to a `std::string`. If it confuses you, I can remove it again.